include/soci/type-conversion.h

```diff
@@ -42,17 +47,15 @@ class conversion_into_type
     typedef typename type_conversion<T>::base_type base_type;
 
     conversion_into_type(T & value)
-        : into_type<base_type>(details::base_value_holder<T>::val_, ownInd_)
+        : into_type<base_type>(details::base_value_holder<T>::val_, details::base_value_holder<T>::ownInd_)
```

Comment:
indeed, I just added the extra namespace qualification to be consistent with rest of the code in this file.
Next commit removes them all